xen/arch/arm/coproc/coproc.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * xen/arch/arm/coproc/coproc.h
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#ifndef __COPROC_H_
+#define __COPROC_H_
+
+#include <xen/sched.h>
```

Comment:
I am afraid but I don't understand you point here.
sched.h is located in include/xen folder.